docs/source/concepts/cl_columns.rst

```diff
@@ -0,0 +1,38 @@
+.. _concepts_cl_columns:
+
+Cleanlab Columns
+****************
+
+Introduction
+============
+
+Cleanlab Studio will run a number of analyses on your dataset to help identify potential data issues. Currently we look for the following issues in datasets:
+- Label issues
+- Outliers
+We will also continue to add new analyses soon. The raw outputs of each analysis will be available to download as Cleanlab columns by calling ``studio.download_cleanlab_columns()``.
+
+For each analysis we will often expose two columns, one being a numeric score between 0 and 1 indicating the quality of a sample for that analysis (lower score means lower quality), and the other being a boolean, which indicates whether the sample is likely to have an issue or not. The different analyses are run separately, and hence they should be independently. For example, samples with label issues are not neccessarily outliers, and vice versa, and it is possible to have a sample with label issue marked as ``False``, but outlier marked as ``True``.
+
+We give more details on the different Cleanlab columns below.
+
+Label Issues
+============
+
+Label quality
+-------------
+The ``label_quality`` column contains a score bounded between 0 and 1. The score is calculated using confident learning. The lower the score of a sample, the more likely it has a label issue.
+
+Issue
+-----
+The ``issue`` column contains a boolean value, with ``True`` indicating that the sample is likely to have a label issue. The value is obtained by thresholding the ``label_quality`` score with confident learning.
```

Comment:
Rename to `label_issue`? Just `issue` will be confusing as we add more analyses (already have > 1).

This is especially important to consider for the future, when we add more analyses: should we have a consistent scheme for naming Cleanlab columns? For the boolean, doing `{name of analysis}_issue` seems good, though `outlier_issue` sounds a bit weird (just `outlier` is better). For the continuous score, using the word "quality" or "score" seems good; choosing one is a bit tricky as well. "label_quality" sounds good, "label_score" doesn't sound quite as good; however, "outlier_quality" doesn't make sense, whereas "outlier_score" is fine.

Reply:
renamed according to our slack discussion:
- add "is_{...}" for boolean
- add "{...}_score" for floats